Source/Core/Map/MapSet.cs

```diff
@@ -3710,7 +3710,36 @@ public static Thing NearestThingSquareRange(ICollection<Thing> selection, Vector
 			// Return result
 			return closest;
 		}
-		
+
+		/// <summary>This finds the sector closest to the specified position.</summary>
+		public static Sector NearestSector(ICollection<Linedef> selection, Vector2D pos) {

```

Comment:
The method doesn't do what the summary and method name suggest. It actually returns the sector at the given position, if there's one. So the method should be renamend to something like `SectorAt`.